apps/studio/src/features/permissions/PermissionsContext.tsx

```diff
@@ -30,11 +30,10 @@ export const PermissionsContext = createContext<ResourceAbility | PureAbility>(
 export const PermissionsProvider = ({
   children,
   siteId,
-  resourceId,
```

Comment:
issue: hmm I feel a bit iffy about this, cos the intent is that we support granular permissions up till the final check where we get the available roles for the user, then we fallback to the site. This removal feels like it is done "somewhere in the middle of the stack".

I thought that this "discarding" of resourceId should still be done in `getResourcePermission`, then all the callers will retain the assumption that it is checking the correct permissions for the specific resource.

Reply:
+1, the code should still retain the architecture/design of granular perms. within the individual checks on backend we can discard the use of resourceId to return site-wide perms